app/modules/individuals/parameters.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+"""
+Input arguments (Parameters) for Individuals resources RESTful API
+-----------------------------------------------------------
+"""
+from flask_restplus_patched import Parameters, PatchJSONParameters
+from . import schemas
+import logging
+import uuid
+
+log = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+def is_valid_uuid(query_id):
+    try:
+        uuid.UUID(str(query_id))
+        return True
+    except ValueError:
+        return False
+
+
+class CreateIndividualParameters(Parameters, schemas.DetailedIndividualSchema):
+    class Meta(schemas.DetailedIndividualSchema.Meta):
+        pass
+
+
+class PatchIndividualDetailsParameters(PatchJSONParameters):
```

Comment:
Currently Encounter patching is implemented so that it requires the user and password in the patch request. I'm not sure if Individual would need this and thinking about it, I'm not entirely certain that Encounter does. 
I suspect this is a JP question

Reply:
I don't think this is necessary since we are checking the current user's permissions for every action. Only case it would allow is for a user to modify an existing encounter if not logged in by verifying with their email and password. Don't think we need that though.